crates/turborepo-ui/src/tui/popup.rs

```diff
@@ -6,19 +6,23 @@ use ratatui::{
     widgets::{Block, List, ListItem, Padding},
 };
 
-const BIND_LIST: [&str; 12] = [
-    "m      - Toggle this help popup",
-    "↑ or j - Select previous task",
-    "↓ or k - Select next task",
-    "h      - Toggle task list",
-    "p      - Toggle pinned task selection",
-    "/      - Filter tasks to search term",
-    "ESC    - Clear filter",
-    "i      - Interact with task",
-    "Ctrl+z - Stop interacting with task",
-    "c      - Copy logs selection (Only when logs are selected)",
-    "u      - Scroll logs up",
-    "d      - Scroll logs down",
+const BIND_LIST: [&str; 16] = [
```

Comment:
We don't need to keep manually updating the length of the underlying array if we only return a slice. Change the type and add a `.as_slice()` call at the end of the array i.e. `];` -> `].as_slice();`
```suggestion
const BIND_LIST: &[&str] = [
```